docs/performance/storage.md

```diff
@@ -171,15 +188,24 @@ DROP TABLE "taxi_deleteme"
 
 > WARNING: Dropping the table deletes the data, make sure that the copy was done correctly.
 
-> INFO: Indexes cannot be re-added after table creation.
+> INFO: Indexes can’t be re-added after table creation.
```

Comment:
```suggestion
:::{attention}
Indexes can’t be re-added after table creation.
:::
```

Reply:
Well its bad wording, they cannot be added after a table is created without them; @hlcianfagna wording is better.